c++/src/capnp/rpc.c++

```diff
@@ -1748,6 +1764,21 @@ private:
     }
 
     RemotePromise<AnyPointer> send() override {
+      if (callBuilder.getIsRealtime()) {
+        // In a situation involving a call being proxied over another connection, it could
+        // happen that send() is called for a realtime stream. In that case, redirect to
+        // sendStreaming().
+        auto streamPromise = sendStreaming().then([]() {
+          auto response = kj::Own<RpcResponse>();
+          auto reader = response->getResults();
+          return Response<AnyPointer>(reader, kj::mv(response));
+        });
+
+        return RemotePromise<AnyPointer>(
+            kj::mv(streamPromise),
+            AnyPointer::Pipeline(getDisabledPipeline()));
+      }
```

Comment:
I am trying to fix the unit test with this, but then I get a segmentation fault [here](https://github.com/capnproto/capnproto/blob/e8c7e3227b8087f0cc2634df12503a023ab11481/c%2B%2B/src/capnp/rpc.c%2B%2B#L2648) on the `connectionState->connection.is<Connected>()` call. Not sure if I am doing something wrong here, or rather [there](https://github.com/capnproto/capnproto/blob/e8c7e3227b8087f0cc2634df12503a023ab11481/c%2B%2B/src/capnp/rpc.c%2B%2B#L3104-L3113) (because it seems like the segmentation fault happens soon after that `return` I make there...).

Reply:
Note: I fixed the segmentation fault that was indeed coming from somewhere else (:tada:), but somehow my unit test is still failing. I'll have to investigate another day, it's getting late here.

Reply:
Argh I don't know. I guess probably `auto response = kj::Own<RpcResponse>();` is the reason for the segmentation fault somewhere down the line.

I'm giving up.